src/main/java/forestry/api/genetics/ISpeciesRoot.java

```diff
@@ -104,7 +162,25 @@
 
 	IGenome templateAsGenome(IAllele[] templateActive, IAllele[] templateInactive);
 
+	/**
+	 * @return A wrapped version of the genome.
+	 *
+	 * @since Forestry 5.8
+	 */
+	default IGenomeWrapper getWrapper(IGenome genome){
+		return null;
+	}
+
 	/* TEMPLATES */
+	/**
+	 * @since Forestry 5.8
+	 */
+	IAlleleTemplateBuilder createTemplateBuilder();
```

Comment:
Without `default` this will break Binnie's. 
If there is no way around it we can just update Binnie's, but then you probably don't need `default` on the `getWrapper` method above this.

Reply:
I removed default